lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -375,12 +379,13 @@ coreApiServer
         , ctx ~ ApiLayer s t k
         )
     => ctx
+    -> StakePoolLayer IO
     -> Server (CoreApi t)
-coreApiServer ctx =
+coreApiServer ctx spl =
     addresses ctx
     :<|> wallets ctx
     :<|> transactions ctx
-    :<|> pools ctx
+    :<|> pools spl
```

Comment:
What about moving `pools` to a separate tree and have something more like:

```hs
server = coreApiServer seqCtx :<|> compatibilityApiServer rndCtx seqCtx :<|> poolServer spl
```

Reply:
Done in `Re-organize API: separate StakePoolApi from CoreApi`